client/settings/settings-tab-panel/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { TabPanel } from '@wordpress/components';
+import { getQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { PaymentMethodsPanel } from '../payment-methods';
+import { PaymentSettingsPanel } from '../payment-settings';
```

Comment:
Rather than using named exports, can these be default exports like the other components?